### DISCUSSION TITLE
Inactive Crowdin translation team

### DISCUSSION THREAD

**Discussion by x9a** [NightscoutFoundation/xDrip#3714] [Q&A]
Few months ago I was interested in contributing to xDrip+ Arabic translation due to my love for open source projects, and the fact that I use this app everyday. When I tried to contribute, I had to fill an application which gets forwarded to Arabic team manager, but months passed by and my application remains on hold. I have tried emailing Crowdin project manager (jamorham) a month ago, but I haven't recieved a response yet. Could anyone guide me how can I start contributing to the translations? Thanks!

**Comment by Angus-repo** [comment]
I, like you, have been waiting for several months and have also contacted Jamorham on Crowdin but haven't received any response.

https://github.com/NightscoutFoundation/xDrip/discussions/3614
https://github.com/NightscoutFoundation/xDrip/discussions/3498

In the end, my approach will be to fork a version of xDrip, work on that translation, and then continually sync the fork with xDrip whenever there are updates.

**Comment by Navid200** [comment]
I have accepted all submissions to join the xDrip project in Crowdin.
Please let me know if you still cannot get in.
